packaging/ubuntu-16.04/rules

```diff
@@ -175,6 +183,7 @@ override_dh_install:
 	install --mode=0644 data/systemd/snapd.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
 	install --mode=0644 data/systemd/snapd.system-shutdown.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
 	$(MAKE) -C cmd install DESTDIR=$(CURDIR)/debian/tmp
+	mv $(CURDIR)/debian/tmp/etc/apparmor.d/usr.lib.snapd.snap-confine $(CURDIR)/debian/tmp/etc/apparmor.d/usr.lib.snapd.snap-confine.real
```

Comment:
The logic here seems a bit curious. The profile was just installed, and we have a dh_apparmor above that already lists the file with the new name. So why is this moving it again?  If this is actually necessary, we need an explanation as otherwise the next editor may undo it.